deploy/60_self_provisioner.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
@aakarshg @jtaleric What are your thoughts on this being in the default deploy directory versus having it be an extra thing you can apply in resources like the backpack role, etc? I'm not necessarily against the self-provisioner role however it would be extending the default permission set so making it more obviously optional might be a good idea.

Reply:
I think its a good idea to move this to resources, so that it's not auto applied. @rvansa can you please move it to resources dir?